tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -912,6 +915,32 @@ async def run_concurrent_ops(self, num_ops, write_weight=.70):
             sent += len(clients)
         return read_count, write_count
 
+    async def run_concurrent_conflict_ops(self, num_ops, write_weight=.70):
```

Comment:
Why do you need this additional method? Isn't the `no_conflicts` flag sufficient to generate conflicts?

Reply:
no_concflict=False **may** generate conflicts.
In this test we want to generate conflict for sure

Reply:
hmm, you are right, but can you at least try to reduce the code duplication between `run_concurrent_ops()` and `run_concurrent_conflict_ops()` ?

Reply:
done

Reply:
done

Reply:
thanks, almost there - see my comment below...